doc/build-instructions/Build_Instructions_V12.md

```diff
@@ -0,0 +1,657 @@
+<!--
+Copyright (c) 2018, 2019 IBM Corp. and others
+
+This program and the accompanying materials are made available under
+the terms of the Eclipse Public License 2.0 which accompanies this
+distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+or the Apache License, Version 2.0 which accompanies this distribution and
+is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+This Source Code may also be made available under the following
+Secondary Licenses when the conditions for such availability set
+forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+General Public License, version 2 with the GNU Classpath
+Exception [1] and GNU General Public License, version 2 with the
+OpenJDK Assembly Exception [2].
+
+[1] https://www.gnu.org/software/classpath/license.html
+[2] http://openjdk.java.net/legal/assembly-exception.html
+
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+Building OpenJDK Version 12 with OpenJ9 (Work in progress, only Linux x86_64 platform verified, depends on https://github.com/eclipse/openj9/pull/3980)
```

Comment:
We need an issue open to track remove the WIP.

Reply:
Can it be https://github.com/eclipse/openj9/issues/3846?

Reply:
Seems a bit risky because we might close it without removing the WIP, but I've made a note in the description.